### ISSUE TITLE
The _ doesn't get hijacked even on packages.

### ISSUE THREAD

**Issue by lsunsi** [stevezhu/meteor-lodash#7]
Right now I have one init.js file in each package with a single _ = lodash line.

I realize the limitations due to meteor core underscore dependency, but the readme clearly says when added in packages, the _ would become lodash.

Is this some trick or am I doing something wrong?

Example: 
This particular piece of code throws a huge
 ```TypeError: Array.prototype.map callback must be a function```
error at compile time, since _ is still underscore and it doesn't support passing keys to a map function.

![screen shot 2015-12-23 at 9 04 10 pm](https://cloud.githubusercontent.com/assets/6089867/11986616/bf0c0842-a9b8-11e5-8441-f85d9d0647a9.png)

**Comment by stevezhu** [comment]
I just tested it out and it was working correctly. You have `api.addFiles('init.js');` before the files you need it in? If it's open source could you link the code? Or I can commit my test code if you're still having trouble.

**Comment by lsunsi** [comment]
It isn't open source, so I actually can't link the code.
In any case, I think I explained myself badly.

If I add the init.js file it works fine, no problem whatsoever.
The thing is I shouldn't need to, right? I mean, the package description says it automatically hijacks the _ variable on packages.

**Comment by stevezhu** [comment]
Oh sorry I haven't worked on this package for so long that I forgot I wrote that so I misunderstood your question too. I just tested that as well though, and it works. Underscore isn't automatically included in packages so normally _ would be defined as lodash, but you probably have another package you are using that calls api.imply('underscore') in which case the behavior is similar to files in an app.
